flutter_appauth/README.md

```diff
@@ -107,6 +119,25 @@ await appAuth.endSession(EndSessionRequest(
 
 The above code passes an `AuthorizationServiceConfiguration` with all the endpoints defined but alternatives are to specify an `issuer` or `discoveryUrl` like you would with the other APIs in the plugin (e.g. `authorizeAndExchangeCode()`).
 
+### Handling errors
+
+Each of these methods will throw exceptions if anything goes wrong. For example:
+
+```dart
+
+try {
+  await appAuth.authorize(...);
+} on FlutterAppAuthPlatformException catch (e) {
+  final FlutterAppAuthPlatformErrorDetails details = e.details;
+  // Handle exceptions based on errors from AppAuth.
+} catch (e) {
+  // Handle other errors.
+}
+
+The `FlutterAppAuthPlatformErrorDetails` object contains all the error information from the underlying platforms, including what's speficied in the [RFC](https://datatracker.ietf.org/doc/html/rfc6749#section-5.2).
```

Comment:
A few comments on this one
- Just noticed this having having left a similar comment below.  IMO, should replace "underlying platforms" with "underlying platform's AppAuth SDK". The name of the class already indicates it's for a single platform that using the plural form could create confusion
- The sentence is rather long too and there's opportunity to break up the part following the comma into its own sentence
- There's a typo "speficied" -> "specified"

Reply:
Done :)